ttadmin/users/migrations/0010_auto_20190119_1828.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
not sure how this additional migration ended up in here? but sure

Reply:
I wondered about that too. idk if it was my configuration on my home server or what. but when i run `manage.py makemigrations` i don't do a lot of questioning of what it creates...